ui/components/multichain/badge-status/badge-status.tsx

```diff
@@ -74,7 +74,7 @@ export const BadgeStatus: React.FC<BadgeStatusProps> = ({
               backgroundColor={badgeBackgroundColor}
               borderRadius={BorderRadius.full}
               borderColor={badgeBorderColor}
-              borderWidth={isConnectedAndNotActive ? 2 : 4}
+              borderWidth={2}
```

Comment:
Before:
<img width="360" alt="Screenshot 2024-12-18 at 15 29 05" src="https://github.com/user-attachments/assets/e6cd73e4-023a-4fc0-b6ff-3bda3382c589" />

After: 
<img width="359" alt="Screenshot 2024-12-18 at 15 08 14" src="https://github.com/user-attachments/assets/a751d7ec-1f51-4f35-a908-205a58bbdebe" />

Reply:
Updated so now the background color of the badge matches the header icon

### After

<img width="383" alt="Screenshot 2024-12-19 at 7 41 54 AM" src="https://github.com/user-attachments/assets/423d3368-5166-4d07-b6ac-95c9010a7bb7" />
<img width="372" alt="Screenshot 2024-12-19 at 7 42 08 AM" src="https://github.com/user-attachments/assets/0e4fde4e-0aed-4f16-a0a8-4a6f608b8091" />